aten/src/ATen/function_wrapper.py

```diff
@@ -543,20 +547,28 @@ def native_translate_formals(argument, option):
         result = [native_translate_formals(argument, option) for argument in result]
         return result
 
+    # this can return multiple return types in a list, e.g. ['Tensor', 'Tensor']
     def native_get_return_types(option):
         ret = option['return']
         if ret['kind'] != 'type':
             raise Exception("native functions only support \'type\' return")
 
-        # can't actually return a TensorList (since it's a reference object)
-        actual_return_type = {'TensorList': 'std::vector<Tensor>'}.get(ret['type'], ret['type'])
-        if actual_return_type == 'Tensor' and option['inplace']:
-            # follow normal ATen convention of returning Tensor & for inplace functions.
-            actual_return_type = 'Tensor &'
-        return [{
-            'type': actual_return_type,
-            'dynamic_type': ret['type'],
-        }]
+        types = [x.strip() for x in ret['type'].split(',')]
```

Comment:
Can we avoid parsing strings inside the code emitter? It is fine to parse stuff in native_parse.py files. For instance here I would expect if we needed multiple returns we would just universally have `option['returns']` here, and have native_parse.py produce a singleton list for most things and a multi-entry when needed.